includes/filters-actions.php

```diff
@@ -178,7 +178,8 @@ function woolab_icdic_checkout_field_process() {
 		if ( $countries->inEurope( $country ) ) {
 
 			// If Validate in VIES
-			if ( woolab_icdic_vies_check() && class_exists('SoapClient') ) {
+			// Slovak DIC cannot (and shouldn't) be validated in VIES
+			if ( woolab_icdic_vies_check() && $country != 'SK' ) {

```

Comment:
Nemazat class_exists

Reply:
[woolab_icdic_vies_check()](https://github.com/vyskoczilova/kybernaut-ic-dic/blob/master/woolab-ic-dic.php#L144) už `class_exists('SoapClient')` obsahuje, je to tam tým pádom duplicitne